modules/_labs/core/react/lib/theming/createCanvasTheme.ts

```diff
@@ -78,8 +79,8 @@ function fillPalette(palette?: PartialCanvasThemePalette): CanvasThemePalette |
   };
 }
 
-export function createCanvasTheme(partialTheme: PartialCanvasTheme): CanvasTheme {
-  const {palette = {}, breakpoints = {}, direction} = partialTheme;
+function calculateCanvasTheme(partialTheme: PartialCanvasTheme): CanvasTheme {
```

Comment:
does the type PartialCanvasTheme mean that the fields that we are passing should be one of CanvasTheme?

Reply:
This type checks any fields that are in common with `CanvasTheme` (i.e. it will complain about `direction: 'foo'`), but it will still allow extra fields.